fv3core/utils/gt4py_utils.py

```diff
@@ -427,9 +427,9 @@ def k_split_run(func, data, k_indices, splitvars_values):
         func(**data)
 
 
-def kslice_from_inputs(kstart, nk, grid):
+def kslice_from_inputs(kstart, nk, grid_indexer):
```

Comment:
If you're changing the type of this, can you please add a type hint so we can identify anywhere the wrong type is being passed?